src/assets/styleJs/events/createEvent.ts

```diff
@@ -0,0 +1,224 @@
+    fontSize: font.fontSize14,
+    "&:nth-child(3)": {
+      marginBottom: margin.marginBottom10,
+    },
+  },
+
+  evaluatorInput: {
+    backgroundColor: color.whiteColor,
+    fontSize: "14px",
+    "&:nth-child(3)": {
+      marginBottom: margin.marginBottom10,
+    },
+    width: "67%",
+    borderRadius: `${borderRedius.top4} ${borderRedius.right0} ${borderRedius.bottom0} ${borderRedius.left4}`,
+  },
+
+  createEventInput: {
+    backgroundColor: color.whiteColor,
+    fontSize: font.fontSize14,
+    "&:nth-child(3)": {
+      marginBottom: margin.marginBottom10,
+    },
+  },
+  evaluatorsArr: {
+    display: "flex",
+    width: "100%",
+    marginTop: margin.marginTop30,
+    marginBottom: margin.marginBottom30
+  },
+  evaluatorsArr1: {
+    display: "flex",
+    width: "100%",
+    marginTop: margin.marginTop30,
+  },
+  evaluateesArr: {
+    display: "flex",
+    width: "100%",
+  },
+  classesHeaderBox: {
+
+  },
+  headerTitle: {
+    display: "flex",
+  },
+
+  headerRegular: {
+    fontWeight: font.fontWeight400,
+    fontSize: font.fontSize16
+
+  },
+  managerText: {
+    fontWeight: font.fontWeight400,
+    fontSize: font.fontSize14
+  },
+
+  infoRootBox:{
+    width: "100%"
+  },
+
+  headerTitleBold: {
+    fontWeight: font.fontWeight700,
+    fontSize: font.fontSize16,
+    marginLeft: margin.marginLeft3
+  },
+  createEventStepper: {
+    marginTop: margin.marginTop25,
+    marginBottom: margin.marginBottom85,
+    display: "flex",
+    justifyContent: "center"
+  },
+  eventTitleText: {
+    fontWeight: font.fontWeight500,
+    fontSize: font.fontSize16,
+    marginRight: margin.marginRight40,
+    whiteSpace: "nowrap",
+  },
+  evaluatorText: {
+    fontWeight: font.fontWeight500,
+    fontSize: font.fontSize16,
+    marginRight: margin.marginRight40,
+  },
+  checkboxManagerBox:{
+    width: "60%",
+    display: "flex",
+    alignItems: "center",
+  },
+  checkbox: {
+    color: "#00e676",
+    padding: 0,
+    marginRight: margin.marginRight8,
+    '& .MuiSvgIcon-root': {
+      fontSize: font.fontSize24 }
+  },
+  nextButton: {
+    width: "146px",
+    height: "36px",
+    marginTop: margin.marginTop15,
+    backgroundColor: color.blueColor,
+    boxShadow: "none"
+  },
+  addButton: {
+    width: "33%",
+    height: "40px",
+    backgroundColor: color.blueColor,
+    borderRadius: `${borderRedius.top0} ${borderRedius.right4} ${borderRedius.bottom4} ${borderRedius.left0}`,
+    boxShadow: "none"
+  },
+  addButtonBox: {
+    display: "flex",
+    width: "100%"
+  },
+  evaluatorNumberErrorText: {
+    marginBottom: margin.marginBottom25,
+    color: color.redColor,
+    ontWeight: font.fontWeight500,
+    fontSize: font.fontSize18
+  },
+
+  stepperStepLabel: {
+    "& > span": {
+      padding : padding.padding0
+    }
+  },
+
+  stepperStack: {
+    width: '70%',
+    padding: padding.padding0,
+  },
+  stepperRoot: {
+    padding: padding.padding0
+  },
+  stepperStep: {
+    padding: padding.padding0
+  }
```

Comment:
same thing for both

do not use 0 as variable

Reply:
done

Reply:
not fixed